### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by ribeiroit**
Hi,

I've added the internationalization support to all strings and already translated the messages to brazilian portuguese. I've been creating an automation to install boh using by puppet available here: https://github.com/ribeiroit/boh-puppet. 

The next steps are creating some functional tests to check translated messages and validate the interfaces.

If you have some considerations, please let me know. 

Thank you, very much.

### CODE REVIEW

File: project/boh/admin.py [hunk 1]
```diff
@@ -143,39 +143,48 @@ class ApplicationAdmin(admin.ModelAdmin):
         (None, {'fields': ['organization', 'name', 'description']}),
         ('Metadata', {
```
Comment (aparsons) [inline project/boh/admin.py]: `_('Metadata')`

File: project/boh/admin.py [hunk 2]
```diff
@@ -143,39 +143,48 @@ class ApplicationAdmin(admin.ModelAdmin):
...
+        (_('Data Classification'), {
             'classes': ['collapse'],
             'fields': ['data_elements']
         }),
-        ('Service Level Agreements', {
+        (_('Service Level Agreements'), {
             'classes': ['collapse'],
             'fields': ['service_level_agreements']
         }),
         ('ThreadFix', {
```
Comment (aparsons) [inline project/boh/admin.py]: `_('ThreadFix')`

**Review by aparsons — CHANGES_REQUESTED** [review]
Thank you for this pull request! I have looked through the changes fully and have a few minor suggestions. After a commit has been made which addresses the two points, I will be happy to merge your contributions. 

Obrigado!

**Review by ribeiroit — COMMENTED** [review]
Agreed!

**Review by aparsons — APPROVED** [review]

**Pull request merged**